### PR TITLE
www-client/qutebrowser: update ebuilds

### PR DESCRIPTION
**Pull request by fturco**

### CODE REVIEW

**Comment by a17r** [comment]
Could you please rebase your branch against the recent changes in tree?

**Comment by fturco** [comment]
Is this OK now? I'm not sure I did everything fine.

**Comment by a17r** [comment]
Not quite, your first commit clearly was(?) about switching from `fdo-mime` to `xdg-utils` (and your git summary should say so, "update ebuilds" is too generic), but it also contains URL changes similar to to the follow-up commits. My suggestion is to either do it all in one commit (for the URL changes are too insignificant, but please do mention them in the git message body) or use `git rebase -i` to undo the URL changes in commit 1 and put them into commit 2. Your call.

Commits 2 and 3 are not in line with our git workflow guidelines, they should be both prefixed with `www-client/qutebrowser:`.

**Comment by fturco** [comment]
Is it OK?

**Comment by a17r** [comment]
Yes, let's wait for the maintainer then, ping @jsbronder

File: www-client/qutebrowser/qutebrowser-9999.ebuild [hunk 1]
```diff
@@ -24,6 +24,7 @@ IUSE="test"
 COMMON_DEPEND="dev-python/setuptools[${PYTHON_USEDEP}]"
 DEPEND="${COMMON_DEPEND}
 	app-text/asciidoc
+	dev-util/source-highlight
```
Comment (jsbronder) [inline www-client/qutebrowser/qutebrowser-9999.ebuild]: This shouldn't be necessary anymore, see     https://github.com/qutebrowser/qutebrowser/commit/dc01b4eaf0e945b131e8f8094d1a7b6a5c873c1c

**Comment by jsbronder** [comment]
I plan on removing 0.9.1 next week.  I left it in to give people time to update to the new release (and update their configs).  Let's just drop the changes to that ebuild.

Thanks for completing the fdo-mime to xdg-utils conversion, didn't realize I missed one.

Drop the re-addition of dev-util/source-highlight and force push this back up and I'll merge.

**Comment by fturco** [comment]
Done

**Comment by jsbronder** [comment]
5db3a238f771 www-client/qutebrowser: use xdg-utils eclass

**Pull request closed without merging**